pyat/at/plot/standalone.py

```diff
@@ -1,6 +1,9 @@
 """AT plotting functions"""
 from __future__ import annotations
 from at.lattice import Lattice, axis_
+from at import RFCavity
+from at.physics.revolution import get_mcf
```

Comment:
A detail about conventions: we assume that the internal organisation of subpackages may change at any time. So here we would write:
```python
from at.lattice import RFCavity
from at.physics import get_mcf
```

Reply:
Good point! I adapted the modules' import accordingly.